.github/workflows/scripts/check_docstrings.py

```
@@ -0,0 +1,773 @@
+    # Initialize key variables
+    func = ""
+    possibles = lines[start:]
+    arguments = []
+    Function = namedtuple("Function", "name arguments")
+    method_keywords = ["self", "cls"]
+
+    # Process the function
+    for line in possibles:
+        if bool(line) is False:
+            continue
+        elif ("'''" not in line) and ('"""' not in line):
```

Ruff (0.11.9)

Unnecessary `elif` after `continue` statement

Remove unnecessary `elif`

(RET507)